2.5/test/run

```diff
@@ -181,6 +181,27 @@ for server in ${WEB_SERVERS[@]}; do
 
   info "All tests for the ${server}-test-app finished successfully."
   cleanup ${server}
+
 done
 
+# test from dockerfile examples
+pushd ${test_dir}
+  info "Check building using a Dockerfile"
+  ct_test_app_dockerfile ../../examples/from-dockerfile/2.5/Dockerfile 'https://github.com/sclorg/rails-ex.git' 'Welcome to your Rails application on OpenShift' app-src
```

Comment:
Please use ${image_dir} or ${test_dir} instead. 
Like here: https://github.com/sclorg/s2i-ruby-container/blob/72669de0cb8a6881118806f6a02636deaaa25311/2.5/test/run#L40

Or wouldn't it work in this case?